engine/class_modules/priest/sc_priest.cpp

```diff
@@ -1367,6 +1367,11 @@ std::unique_ptr<expr_t> priest_t::create_expression( util::string_view expressio
               };
             } );
           }
+          else if ( splits[ 2 ] == "down" )
+          {
+            bool down = get_cooldown( pet->name_str )->down();
+            return make_fn_expr( expression_str, [ pet, down ] { return down; } );
```

Comment:
As far as I can tell, this is basically just a constant expression that will always return false.

I *think* what you actually want to do here is something along the lines of
```cpp
cooldown_t* cd = get_cooldown( pet->name_str );
return make_fn_expr( expression_str, [ cd ] { return cd->down(); } );
```

Reply:
Hm I can try that instead. But the condition is returning true in my testing I did